cmd/tempo/app/modules.go

```diff
@@ -390,6 +420,9 @@ func (t *App) setupModuleManager() error {
 	mm := modules.NewManager(log.Logger)
 
 	mm.RegisterModule(Server, t.initServer, modules.UserInvisibleModule)
+	if t.cfg.InternalServer.Enable {
```

Comment:
This if statement needs to be removed. Even if the internal server is disabled we still need to register the module b/c of the changes just made.